src/elements/content-sidebar/stories/BoxAISidebar.stories.tsx

```diff
@@ -15,3 +13,5 @@ export default {
         token: global.TOKEN,
     },
 };
+
+export const basic = {};
```

Comment:
What is the reason for moving this line below default export?

Reply:
just to reiterate, the `default` export is usually on the bottom, so the `basic` export should be moved above it